subsys/net/lib/lwm2m/Kconfig

```diff
@@ -110,6 +110,13 @@ config LWM2M_QUEUE_MODE_UPTIME
 	  defaults to 93 seconds, see RFC 7252), it does not forbid other
 	  values though.
 
+config LWM2M_QUEUE_MODE_NO_MSG_BUFFERING
+	bool "Disable buffering notifications and messages from SEND operation"
+	select EXPERIMENTAL
+	depends on NET_SOCKETPAIR
+	help
+	  Messages are sent right away instead of waiting for next registration update.
+
```

Comment:
```suggestion
config LWM2M_QUEUE_MODE_NO_MSG_BUFFERING
	bool "Disable buffering notifications and messages on queue mode"
	select EXPERIMENTAL
	depends on NET_SOCKETPAIR
	help
	  Messages are sent right away instead of triggering a registration update.
	  This might not be supported on all servers.

```